test/test_utils.py

```diff
@@ -156,6 +156,37 @@ def multi_build(request, recipes_fixture, config_fixture):
             ensure_missing(pkg)
 
 
+@pytest.fixture(scope='module', params=PARAMS, ids=IDS)
```

Comment:
Doesn't this have to get called from a test like this in order to actually test it? https://github.com/bioconda/bioconda-utils/blob/b007d34e6c723f7f9d6fcb5a6f58e072d4618cdf/test/test_utils.py#L228